src/xdotoolify.js

```diff
@@ -230,7 +230,18 @@ _Xdotoolify.prototype.checkUntil = function(f, ...rest) {
   });
   return this;
 };
-_Xdotoolify.prototype.mousemove = function(selector, relpos, twoStep, timeout) {
+_Xdotoolify.prototype.checkNothing = function() {
+  this._addOperation({
+    type: 'skip'
```

Comment:
can we add a no-op checkUntil instead of calling it skip? It is negligibly more expensive but much more obvious.

Reply:
Now checkNothing uses a callback that immediately returns true.